### DISCUSSION TITLE
Images de Pasteles

### DISCUSSION THREAD

**Discussion by jmlimas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#298] [General]
Hola  compañeros  de  donde  de  recomiendan  bajar a fotos  de  pasteles de calidad conocen  alguna  pagina de  imagenes  gratis?

**Comment by manusanabria** [comment]
Pixabay

**Reply by jmlimas** [reply]
Gracias,  manusanabria

**Comment by JuanSalvador73** [comment]
Ve a https://unsplash.com y ahí tecleas pics free. De ahí bajé un buen, y a excelente resolución.
